cypress/e2e/home.cy.ts

```diff
@@ -122,3 +122,69 @@ describe(" Footer: UI and functionality:", () => {
       .contains("Rocket wallet 2023");
   });
 });
+
+describe(" Wallet: UI and functionality:", () => {
```

Comment:
There's a blank space here.

Reply:
solved in commit 511e7114d29729205dbe09561458400491ef7083.